composer.json

```diff
@@ -37,7 +39,7 @@
         ]
     },
     "extra": {
-        "_ezplatform_branch_for_behat_tests": "master",
+        "_ezplatform_branch_for_behat_tests": "EZP-28032-rest-common",
         "branch-alias": {
             "dev-master": "0.10.x-dev",
```

Comment:
It also requires version bump, it's no longer technically possible to support multiple versions of ezpublish-kernel with these changes.
I guess `0.11.x-dev` should be fine // ping @andrerom

Reply:
Yup, that's what I wrote in PR description but was opting for 1.0 to be somewhat consistent with other packages, but I have no strong opinion on that.

Reply:
3.x version can aim to be 1.0 in the end if we'd like, but there are a tons of code to remove and adapt to use FOSHttpCache 2.x before we can make that stable.

Reply:
Ok, so I will go with 0.11 for now

Reply:
I need to ask, out of curiosity - where will be 0.10 used? 2.5 uses 0.9, 0.10 was never released, do we really need 0.11?

Reply:
Oh, well, I think we all didn't noticed that `0.10` was created for `3.0`

Reply:
> I need to ask, out of curiosity - where will be 0.10 used? 2.5 uses 0.9, 0.10 was never released, do we really need 0.11?

@mnocon :trophy:

Reply:
We could go for 1.0.x-dev for 3.0, couldn't we?

Reply:
I think we should, but it would make most sense once we make it FOSHttpCache 2.x compatible, which we also planned to do anyway.

Reply:
We HAVE TO go for `1.0` with eZ Platform 3 since it won't be backwards compatible.

Reply:
> We HAVE TO go for 1.0 with eZ Platform 3 since it won't be backwards compatible.

You don't have to _(0.x versions are all semantically major versions), but I prefer you go for 1.0.x-dev here for 3.x like @lserwatka prefer too. As it allows us bump to 0.10 and higher for 1.13/2.5 in case we need to introduce features/feature-changes to solve bugs, like for instance shorten tag length. .. :)

Reply:
Since it's already configured to be 3.0 compatible only, let's keep it as is and bump to 1.0 once we make FOSHttpCache 2.x compatibility.